tests/test_sets.py

```diff
@@ -192,7 +192,7 @@ def test_set_get_draft(client):
         responses.GET,
         client.get_url("SET_VERSION", "GET", "draft", {"id": 1}),
         body=sets_new_draft_good_simulated_response,
-        status=201,
+        status=201,     # Should be 307
```

Comment:
A 307 should come back from this request, but setting that breaks the test
And I don't understand how the test framework really works, so not clear how to fix